src/visitor/FindAnyExprVisitor.cpp

```diff
@@ -0,0 +1,149 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "visitor/FindAnyExprVisitor.h"
+
+namespace nebula {
+namespace graph {
+
+FindAnyExprVisitor::FindAnyExprVisitor(const std::unordered_set<Expression::Kind> &kinds)
+    : kinds_(kinds) {
+    DCHECK(!kinds.empty());
+}
+
+void FindAnyExprVisitor::visit(TypeCastingExpression *expr) {
+    findExpr(expr);
+    if (found_) return;
+    expr->operand()->accept(this);
+}
+
+void FindAnyExprVisitor::visit(UnaryExpression *expr) {
+    findExpr(expr);
+    if (found_) return;
+    expr->operand()->accept(this);
+}
+
+void FindAnyExprVisitor::visit(FunctionCallExpression *expr) {
+    findExpr(expr);
+    if (found_) return;
+    for (const auto &arg : expr->args()->args()) {
+        arg->accept(this);
+        if (found_) break;
```

Comment:
better to use `return`, keep the same as under code.

Reply:
fixed